src/codegen/function_body_chunk.rs

```diff
@@ -1338,15 +1366,15 @@ fn add_chunk_for_type(
                 crate::codegen::trampoline_from_glib::from_glib_xxx(par.transfer, true);
 
             let type_name;
-            if ty_name == "GString" {
+            if ty_name.ends_with("GString") {
                 if *nullable {
                     if par.conversion_type == ConversionType::Borrow {
-                        type_name = String::from(": Borrowed<Option<GString>>");
+                        type_name = String::from(": Borrowed<Option<glib::GString>>");
                     } else {
-                        type_name = String::from(": Option<GString>");
+                        type_name = String::from(": Option<glib::GString>");
```

Comment:
Always `glib::`? Might also have to be `crate::` if this ever gets called from inside glib :)

Reply:
It seems not because I never encountered the case but it'd be better to not do it by default.

Reply:
But just a few lines above we have this check:

```rust
library::Type::Alias(_) if ty_name == "glib::Pid" => {
```

So I wonder if it's worth it to change it?

Reply:
How do you mean?